aten/src/ATen/native/native_functions.yaml

```diff
@@ -1347,11 +1347,6 @@
     CPU: native_group_norm_backward
     CUDA: native_group_norm_backward
 
-- func: quantized_group_norm(Tensor input, int num_groups, Tensor? weight, Tensor? bias, float eps, float output_scale, int output_zero_point) -> Tensor
```

Comment:
could you remove quantized_batch_norm as well?

Reply:
np.  We can do that as a separate PR, since it's not related to this diff, technically.